### PR TITLE
Dark mode blue

### PR DESCRIPTION
**Pull request by ecampi**
**Issue**

Closes issue #286 

**Photos**
Before:
image

After:
image

I made the colors for the buttons and links lighter so they would be easier to see

### CODE REVIEW

File: src/web/src/assets/dark.scss [hunk 1]
```diff
@@ -53,6 +53,28 @@
     }
   }
 
+  .nav-link {
```
Comment (marchdan) [inline src/web/src/assets/dark.scss:56]: Could you make a var(...) for these so they fit the styling of the rest of the file and can be more easily reused?

**Comment by marchdan** [comment]
@ecampi Can you make a var(...) for the new colors so that this addition fits the rest of the dark.scss format?

**Comment by YuetingLiao** [comment]
Should we color the "Spring 2021" instead of removing it?

**Comment by marchdan** [comment]
> Should we color the "Spring 2021" instead of removing it?

I see no issue with the current coloring of the semester name in the header @YuetingLiao

**Comment by YuetingLiao** [comment]
> > Should we color the "Spring 2021" instead of removing it?
> 
> I see no issue with the current coloring of the semester name in the header @YuetingLiao

I mean the second image removed the title of semesters, not sure if this is the change we want.

**Comment by marchdan** [comment]
> > > Should we color the "Spring 2021" instead of removing it?
> > 
> > 
> > I see no issue with the current coloring of the semester name in the header @YuetingLiao
> 
> I mean the second image removed the title of semesters, not sure if this is the change we want.

Its not, we want to leave the name there, but the color doesn't need to leave

**Comment by ecampi** [comment]
> @ecampi Can you make a var(...) for the new colors so that this addition fits the rest of the dark.scss format?

Ok I just put in the vars and commited it here

**Comment by marchdan** [comment]
@ecampi Please resolve the merge conflicts

**Review by RichtXO — APPROVED** [review]

**Review by IkeLyons — APPROVED** [review]

**Pull request merged**